manifests/c/Carvel/ytt/0.47.0/Carvel.ytt.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.3.2 $debug=QUSU.CRLF.7-4-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Carvel.ytt
+PackageVersion: 0.47.0
+PackageLocale: en-US
+Publisher: Carvel
+PublisherUrl: https://carvel.dev
+PublisherSupportUrl: https://carvel.dev/community/
+# PrivacyUrl: 
+Author: Carvel Authors
+PackageName: ytt
+PackageUrl: https://carvel.dev/ytt/
+License: Apache 2.0
+LicenseUrl: https://github.com/carvel-dev/ytt/blob/v0.45.0/LICENSE
```

```suggestion
LicenseUrl: https://github.com/carvel-dev/ytt/blob/develop/LICENSE
```

Use the default branch link instead of a versioned one, so that it can be easily carried forward for manifest updates.